### ISSUE TITLE
WCAG 2.1.1: Elements that have scrollable content should be accessible by keyboard (.fixedHight200.marginBottom5.form-group:nth-child(1))

### ISSUE THREAD

**Issue by BorisGuan** [BorisGuan/MDSIssuesRepro#4]
#### Issue
`Elements that have scrollable content should be accessible by keyboard` ([scrollable-region-focusable](https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable))
#### Target application
[Explore - Master Data Services](http://eims-mds-01/MDS/Explore/Entity?MID=5985e0f9-1f8c-4c87-9ebe-01ad857d078e&VID=47282893-b80e-4970-8951-ca661904c43b&EID=da91b8b0-c520-4260-a17c-f384b93cfe4e)
#### Element path
.fixedHight200.marginBottom5.form-group:nth-child(1)
#### Snippet
`<div class="form-group fixedHight200 marginBottom5">`
#### How to fix
    Fix any of the following:
      Element should have focusable content
      Element should be focusable
#### Environment
Chrome version 88.0.4324.104
This accessibility issue was found using Accessibility Insights for Web 2.23.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
